Bang/Game/Gameplay.cs

```diff
@@ -150,5 +150,61 @@ public void ChooseCard(BangGameCard card, Player player)
         {
             state = state.ApplyReplyAction(player, card);
         }
+
+        public void StartNextPlayerTurn()
+        {
+            SetNextPlayer();
+
+            var dynamiteCard = PlayerTurn.PlayerTablet.ActiveCards.FirstOrDefault(c => c == new DynamiteCardType());
```

Comment:
`PlayerTurn.ActiveCards.FirstOrDefault(c => c == new DynamiteCardType())` would be shorter

Reply:
done